src/main/java/eu/xlime/kafka/rdf/Summary.java

```diff
@@ -0,0 +1,53 @@
+package eu.xlime.kafka.rdf;
+
+import java.util.Map;
+import java.util.Date;
+import java.util.Calendar;
+
+public class Summary {
+	private String consumerId;
+	private Map<String,Long> annotations;
+	private long processed;
+	private Date date; 
+	
+	public Summary(){
+		date = Calendar.getInstance().getTime();
+	}
+
+	public String getConsumerId() {
+		return consumerId;
+	}
+
+	public void setConsumerId(String consumerId) {
+		this.consumerId = consumerId;
+	}
+
+	public Map<String,Long> getAnnotations(){
+		return annotations;
+	}
+	
+	public void setAnnotations(Map<String,Long> annotations){
```

Comment:
Clients of this class need to set the map of annotations, which can limit how the class can be used. 

E.g. suppose we have a hierachy of Class A and subclass B. Class A generates a Summary and sets some of the counters and leaves it up to B to generate some other counters.  It would be better to provide methods like `setCounter(name, count)` and `addCounters(Map<String, Long>)`.

I.e.
 *  make the Map internal and private to the Summary class (it can be a hashMap). `getAnnotations` would return an (immutable) copy of this internal map.
 * remove method `setAnnotations`
 * add methods `setCounter' and 'addCounters'